modules/exploits/linux/redis/redis_unauth_exec.rb

```diff
@@ -79,7 +79,19 @@ def check
       report_redis(redis_version)
     end
 
-    Exploit::CheckCode::Vulnerable
+    unless redis_version
+      print_error("Cannot retrieve redis version, please check it manually")
```

Comment:
Minor nitpick for next time: single quotes, please. You're consistent elsewhere. :)

Reply:
Hehe, I fixed it here: https://github.com/rapid7/metasploit-framework/commit/42c8420f5ab2086fb2ab41a5c8448199fb6cbcd5.